notesapi/v1/tests/test_views.py

```diff
@@ -959,6 +959,26 @@ def test_pagination_page_start(self, page, start, previous_page, next_page):
             start=start
         )
 
+    @ddt.unpack
+    @ddt.data(
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"محمد عمار", "tags": [u"عمار", u"Muhammad", u"محمد"]},
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"محمد", "tags": [u"محمد", u"Muhammad"]},
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"عمار", "tags": [ u"ammar", u"عمار"]},
+        {"text": u"Muhammad Ammar", "search": u"عمار", "tags": [ u"ammar", u"عمار"]},
+        {"text": u"محمد عمار", "search": u"Muhammad", "tags": [ u"Muhammad", u"عمار"]}
+    )
+    @unittest.skipIf(settings.ES_DISABLED, "MySQL cannot do highlighting")
```

Comment:
@robrap yes, it is running. I have increased the verbosity so now test names are show on console. You can check at https://travis-ci.org/edx/edx-notes-api/builds/109792097. Search `test_search_unicode_text_and_tags` in last two build on the above link.

I don't see any scenario in which wrong data will be passed to frontend. @nedbat Please share your thoughts.